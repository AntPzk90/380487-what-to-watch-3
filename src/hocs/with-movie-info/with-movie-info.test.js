import React from 'react';
import renderer from 'react-test-renderer';
import withMovieInfo from './with-movie-info.jsx';


it(`SnapshotTest withMovieInfo`, () => {

  const tree = renderer
  .create(
      <withMovieInfo
        onTabClick={() => {}}
        activePage={`overview`}
        />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
