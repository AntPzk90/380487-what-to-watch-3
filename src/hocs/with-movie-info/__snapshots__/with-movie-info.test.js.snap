// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest withMovieInfo 1`] = `
<withMovieInfo
  activePage="overview"
  onTabClick={[Function]}
/>
`;
